The Descent-bab978/Program.c

```diff
@@ -1,52 +1,80 @@
-/* # Descent into CodeChat */
-/* ## Code explanation */
-/*  The *while(1)* loop is what runs the game.
-    Each iterative run of the loop is a turn in the game
-    in which there will be an *input* of the mountains and the *output* to fire at a mountains.
-    The inputs are updated per run of the loop */
+/* # Descent into CodeChat
```

Comment:
Delete this from your PR -- all I want is the modifications to your code.

Reply:
Deleted.